unit-test is failing  for startcmd

--- FAIL: TestStartAriesDRequests (2.06s)
##[error]    start_test.go:338: timeout: server is not available

FAIL github.com/hyperledger/aries-framework-go/cmd/aries-agentd/startcmd 5.369s

--- a/cmd/aries-agentd/startcmd/start_test.go
+++ b/cmd/aries-agentd/startcmd/start_test.go
@@ -114,7 +114,7 @@ func TestStartAriesDRequests(t *testing.T) {
 }
 
 func listenFor(host string) error {
-	timeout := time.After(2 * time.Second)
+	timeout := time.After(5 * time.Second)
 	for {
 		select {
 		case <-timeout:

